include/proc.h

```diff
@@ -104,10 +104,3 @@ struct proc {
 //   original data and bss
 //   fixed-size stack
 //   expandable heap
-
```

Comment:
We should keep the struct definition in the .h file since it is needed in two places.